Tests/CommonHelpers/Acts/Tests/CommonHelpers/BenchmarkTools.hpp

```diff
@@ -299,7 +300,8 @@ struct MicroBenchmarkResult {
   //
   Duration runTimeRobustStddev() const {
     auto [firstq, thirdq] = runTimeQuartiles();
-    return (thirdq - firstq) / (2. * std::sqrt(2.) * 0.4769362762044698733814);
```

Comment:
What's the 0.4769.... do you have an idea ?

Reply:
I was also quite confused, but figured out after some trials, that it is `invErf(0.5)`, as mentioned function documentation.